agents/skala-r/fence_skalar.py

```diff
@@ -0,0 +1,222 @@
+#!@PYTHON@ -tt
+# -*- coding: utf-8 -*-
+
+import sys
+import atexit
+sys.path.append("@FENCEAGENTSLIBDIR@")
+from fencing import *
+from fencing import fail, EC_STATUS, EC_LOGIN_DENIED, EC_INVALID_PRIVILEGES, run_delay
+import requests
+import ast
+import urllib3
+import json
+import logging
+
+from requests.exceptions import ConnectionError
+
+urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)
+
+cookie = None
+
+RELEASE_VERSION="1.1"
```

Comment:
You can remove these, as we've removed them from the other agents: https://github.com/ClusterLabs/fence-agents/pull/131/files.

Reply:
[root@node01 ~]# fence_skalar -V
Traceback (most recent call last):
  File "/usr/sbin/fence_skalar", line 231, in <module>
    main()
  File "/usr/sbin/fence_skalar", line 202, in main
    show_docs(options, docs)
  File "/usr/share/fence/fencing.py", line 753, in show_docs
    print(__main__.RELEASE_VERSION, __main__.BUILD_DATE)
AttributeError: 'module' object has no attribute 'RELEASE_VERSION'

Reply:
> You can remove these, as we've removed them from the other agents: https://github.com/ClusterLabs/fence-agents/pull/131/files.

[root@node01 ~]# fence_skalar -V
Traceback (most recent call last):
  File "/usr/sbin/fence_skalar", line 231, in <module>
    main()
  File "/usr/sbin/fence_skalar", line 202, in main
    show_docs(options, docs)
  File "/usr/share/fence/fencing.py", line 753, in show_docs
    print(__main__.RELEASE_VERSION, __main__.BUILD_DATE)
AttributeError: 'module' object has no attribute 'RELEASE_VERSION'

Reply:
Oh. It seems like you have an old version of fencing.py on your nodes.

Reply:
> Oh. It seems like you have an old version of fencing.py on your nodes.
Yes, already realized it;) Unfortunately this script should works in old env too.
But I'm going to make what you have advised.

Reply:
Thanks.

My mistake. You can also remove the 2 lines below it.

Reply:
Ready for the review. Removed RELEASE_VERSION + a few improvement related to ssl

Reply:
> Thanks.
> 
> My mistake. You can also remove the 2 lines below it.

Done.